src/components/select/simple-select/simple-select.spec.js

```diff
@@ -130,7 +130,7 @@ describe("SimpleSelect", () => {
 
     assertStyleMatch(
       {
-        paddingRight: "0",
+        paddingRight: "40px",
```

Comment:
I think it would be good to have a test for all the potential paddings defined in `getInputIconWidth`